Plugins/Flow.Launcher.Plugin.Explorer/Search/Everything/SortOption.cs

```diff
@@ -27,7 +27,6 @@ public enum SortOption : uint
         ATTRIBUTES_DESCENDING = 16u,
         FILE_LIST_FILENAME_ASCENDING = 17u,
         FILE_LIST_FILENAME_DESCENDING = 18u,
-        RUN_COUNT_ASCENDING = 19u,
```

Comment:
well...it is silly though; Will someone want never clicked result???🤣

Reply:
exactly! that's why i removed that option.